Nos/Views/Profile/ProfileHeader.swift

```diff
@@ -60,42 +60,57 @@ struct ProfileHeader: View {
                             )
                     }
                     VStack(alignment: .leading, spacing: 3) {
-                        Spacer()
-
+                        // Name
                         Button {
                             showingBio = true
                         } label: {
                             Text(author.safeName)
                                 .lineLimit(1)
-                                .font(.clarity(.bold, textStyle: .title3).weight(.semibold))
+                                .truncationMode(.tail)
+                                .font(.title3.weight(.bold))
                                 .foregroundColor(Color.primaryTxt)
```

Comment:
Looks to me like this could use a small amount of padding on the top to match the design. Or we could pad the top of the VStack, but either way a little padding should make this look like the Figma file.